client/my-sites/earn/home.tsx

```diff
@@ -152,14 +153,14 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 					},
 			  };
 		const title = hasConnectedAccount
-			? translate( 'Manage Recurring Payments' )
+			? translate( 'Manage recurring payments' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.